pymilvus/client/grpc_handler.py

```diff
@@ -653,6 +654,19 @@ def search_with_expression(self, collection_name, data, anns_field, param, limit
                                                      partition_names, output_fields, **_kwargs)
         _kwargs.pop("schema")
         _kwargs["auto_id"] = auto_id
+        # tag edit
+        round_decimal = param.pop("round_decimal",3)
+        if not isinstance(round_decimal,(int,str)):
```

Comment:
The two parameters should be separated by a space. Other places have the same problem.